Policies/Kubernetes/fsgroup/azurepolicy.json

```diff
@@ -0,0 +1,79 @@
+{
+  "properties": {
+    "displayName": "Control allowed fsgroup on pods in a Kubernetes Cluster",
+    "policyType": "Custom",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy controls valid ranges for FsGroup (File System Group) on volumes in a Kubernetes cluster. For instructions on using this policy, please visit https://aka.ms/kubepolicydoc.",
+    "metadata": {
+      "version": "1.0.0-preview",
+      "category": "Kubernetes PSP",
+      "preview": true
+    },
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
+          "description": "Enable or disable the execution of the policy"
+        },
+        "allowedValues": [
+          "enforceOPAConstraint",
+          "disabled"
+        ],
+        "defaultValue": "enforceOPAConstraint"
+      },
+      "excludedNamespaces": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Namespace exclusions",
+          "description": "List of Kubernetes namespaces to exclude from policy evaluation. Providing a value for this parameter is optional."
+        },
+        "defaultValue": []
+      },
+      "fsgroupRule": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Rule for a pod to confirm to FsGroup ranges.",
```

Comment:
nit: FSGroup as it is File System Group. Comment applicable in other places as well where 'FsGroup' is used.